ui/elements/TabList/TabList.test.tsx

```diff
@@ -236,13 +236,13 @@ describe('TabList', () => {
         });
 
         it('renders with border variant explicitly set', () => {
-            render(<TabList variant="border">{defaultChildren}</TabList>);
+            render(<TabList variant="bordered">{defaultChildren}</TabList>);
 
             const tablist = screen.getByRole('tablist');
             expect(tablist).toBeInTheDocument();
 
             // Should render the indicator for border variant
-            const indicator = tablist.querySelector('.tab-indicator');
+            const indicator = tablist.querySelector('.ns-tab-indicator');
```

Comment:
if className was added solely for usage within test-case, then we can use data-testid (and use selector getByTestId) for this instead

hard-coded className should be given only for allowing external customization

if there is scope for customzation of this tab indicator (need to document that too) then we can keep this className, otherwise lets remove className here too in favor of data-testid

for example, checkout the story for Accordion component, usagae of `ns-something` classname has been documented in this story https://grauity.newtonschool.co/?path=/docs/elements-accordion--docs